week-5/F-webchat/exercise-1.js

```diff
@@ -33,5 +33,19 @@ When you open index.html in your browser, it should display the existing message
 
 */
 
-
-// Write your code here
\ No newline at end of file
+// Write your code here
+function messages() {
+  fetch("https://codeyourfuture.herokuapp.com/api/messages")
+    .then(function(x) {
+      return x.json();
+    })
+    .then(function(y) {
```

Comment:
Always try to have descriptive names for parameters. So instead of x and y, it might be better named "response" and "responseJson", to keep the code more readable.